AdyenActions/Components/3DS2/ThreeDS2Component.swift

```diff
@@ -58,15 +58,28 @@ public final class ThreeDS2Component: ActionComponent {
             /// The Apple registered development team identifier.
             public let appleTeamIdentifier: String
 
+            /// The configuration for Delegated Authentication Component style
+            public let delegatedAuthenticationComponentStyle: DelegatedAuthenticationComponentStyle
+
+            /// The localization parameters, leave it nil to use the default parameters.
+            public let localizationParameters: LocalizationParameters?
+
             /// Initializes a new instance.
             ///
             /// - Parameter localizedRegistrationReason: The localized reason string show to the user while registration flow.
             /// - Parameter localizedAuthenticationReason: The localized reason string show to the user while authentication flow.
             /// - Parameter appleTeamIdentifier: The Apple registered development team identifier.
-            public init(localizedRegistrationReason: String, localizedAuthenticationReason: String, appleTeamIdentifier: String) {
+            /// - Parameter delegatedAuthenticationComponentStyle: The delegated authentication component style.
+            public init(localizedRegistrationReason: String,
+                        localizedAuthenticationReason: String,
+                        appleTeamIdentifier: String,
+                        delegatedAuthenticationComponentStyle: DelegatedAuthenticationComponentStyle = .init(),
+                        localizationParameters: LocalizationParameters? = nil) {
```

Comment:
parameter is missing in the function documentation

Reply:
Done. Thanks ✅